README.md

```diff
@@ -13,11 +13,25 @@ Every raw message contains important parts:
 
 session alias, direction and sequence number are a compound unique identifier of raw messages within th2
```

Comment:
**session alias, direction and sequence number** are a compound unique identifier of raw messages within th2 (I suggest just to mark in bold the compound unique identifier)